internal/commands/app/create_test.go

```diff
@@ -335,75 +330,78 @@ Check out your app: cd ./test-app && realm-cli app describe
 			procedure(console)
 		}()
 
-		var createdApp realm.App
-		rc := mock.RealmClient{}
-
-		zipPkg, err := zip.OpenReader("testdata/project.zip")
-		assert.Nil(t, err)
-		defer zipPkg.Close()
```

Comment:
looks like we dropped the `defer Close()` on `defaultBackendZipPkg` (the same I think goes for `frontendZipPkg` in this file too?)